Makefile

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
I did rebase :) It's still here on the master: https://github.com/appcelerator/amp/blob/master/Makefile#L1